### PR TITLE
mod mail

### PR DESCRIPTION
**Pull request by TheColaber**
wip pls give me time to finish

### CODE REVIEW

File: commands/modmail.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const { MODMAIL_CHANNEL } = process.env;
+if (!MODMAIL_CHANNEL) throw new Error("MODMAIL_CHANNEL is not set in the .env.");
+
+/** @type {import("../types/command").default} */
+const info = {
+	data: new SlashCommandBuilder()
+		.addSubcommand((subcommand) =>
+			subcommand
+				.setName("close")
+				.setDescription("(Mods Only) Send a message in a modmail thread"),
```
Comment (RedGuy12) [inline commands/modmail.js:14]: update desc

File: commands/modmail.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			await starter.edit({
+				embeds: [
+					{
+						title: "ModMail Ticket Closed",
+						description: starter.embeds[0]?.description || "",
+						color: 0x00ff00,
+					},
+				],
+			});
+			interaction.channel.delete("Closed by " + interaction.user.username);
```
Comment (RedGuy12) [inline commands/modmail.js:44]: dont delete just archive and lock

Comment (TheColaber) [reply commands/modmail.js:44]: @RedGuy12 archive and lock still allows mods to surpass it

Comment (RedGuy12) [reply commands/modmail.js:44]: mods dont have manage threads, only admins do

I see your point tho

the thing is, it'd be good to have a record of modmails

**Comment by TheColaber** [comment]
this needs to be tested with all possible conditions before merging

**Comment by retronbv** [comment]
make it draft

**Pull request merged**